OpenRA.Mods.Cnc/FileSystem/BigFile.cs

```diff
@@ -48,6 +48,7 @@ public BigFile(Stream s, string filename)
 					// and we don't have to try seeking there since the entries typically start next in EA's .big files.
 					s.ReadUInt32();
 
+					index = new Dictionary<string, Entry>((int)entryCount);
```

Comment:
Sanity check was also already missing here.

Reply:
As above, we should tackle separately.